tests/unit/schema/schemaManager/schemaManger.spec.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { SchemaManager } from '../../../../src/schema/models/schemaManager';
+
+let schemaManager: SchemaManager;
+
+describe('SchemaManager', () => {
+  let getSchema: jest.Mock;
+  let getSchemas: jest.Mock;
+
+  const schema = {
+    name: 'system1',
+    createdAt: '111',
+    updatedAt: '1111',
+    mapping: {
+      externalKey1: 'osmKey1',
+      externalKey2: 'osmKey2',
+    },
+  };
+
+  beforeEach(() => {
+    getSchemas = jest.fn().mockReturnValue([schema]);
+    getSchema = jest.fn().mockReturnValue(schema);
+    schemaManager = new SchemaManager({ getSchemas, getSchema });
+  });
+
+  afterEach(() => {
+    getSchemas.mockReset();
+  });
+
+  describe('#getSchemas', () => {
```

Comment:
add a check for the instance of `schema` for items in the array

Reply:
schema is a ts interface not class. jest cannot test intefaces